Cartridges/int_checkoutcom_sfra/cartridge/client/default/js/product/applePayDetail.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
```

Comment:
i don't think we should add this file to our integration. Please can you look into removing this file also.